kantox-obfuscator.gemspec

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |s|
+  s.name        = 'kantox-obfuscator'
```

Comment:
I would call it just `obfuscator`, it doesn't have anything Kantox-specific from what I can gather.

Reply:
There is already a gem called obfuscator, very old. I want to avoid name clashes

Reply:
Maybe `obfuscaty` or something else? :)

Reply:
“enredo”

FWIW: my most helpful and hi-quality elixir packages are called “peluquero” and “tarearbol” respectively. And I know there is a tendency to call elixir packages “packagenamEX,” but I am not as dumb as to follow tendencies; we were born to build our own rules.